data/views/redfish-1.0/redfish.1.0.0.taskservice.1.0.0.json

```diff
@@ -1,7 +1,7 @@
 {
-    "@odata.context" : "<%= basepath %>/$metadata/TaskService",
+    "@odata.context" : "<%= basepath %>/$metadata/TaskService.TaskService",
```

Comment:
Change the "/" to "#" in odata.context after $metadata
as "/$metadata#TaskService.TaskService"